source/frontend/src/components/Table/Table.tsx

```diff
@@ -295,7 +295,7 @@ export const Table = <T extends IIdentifiedObject, TFilter extends object = {}>(
   const manualSortBy = !!externalSort || props.manualSortBy;
   const totalItems = externalTotalItems ?? data?.length;
   const pageCount =
-    externalPageCount ?? internalPageSize > 0 ? Math.ceil(totalItems / internalPageSize) : 0;
+    externalPageCount ?? (internalPageSize > 0 ? Math.ceil(totalItems / internalPageSize) : 0);
```

Comment:
these brackets are actually required, otherwise the pageCount does not update properly when the page size changes.